doc/srv6/srv6-hld-v19.md

```diff
@@ -0,0 +1,606 @@
+
+
+
+
+
+
+
+
+
```

Comment:
please remove these extra lines. Also i'm not sure why file has name as `srv6-hld-v19`. The version in file is 0.2. Lets not have version in the filename

Reply:
removed, thanks~